src/Augurk.CommandLine/Commands/PublishCommand.cs

```diff
@@ -252,6 +252,7 @@ private Feature ParseFeatureFile(string featureFile)
         {
             using (var reader = new StreamReader(featureFile))
             {
+                Stack<string> directories = new Stack<string>();
```

Comment:
Since we're only looping over one file at the same time, wouldn't simply storing the current directory in a local variable suffice, rather than creating a Stack which will only ever contain one item?